src/frontend/context/ObservationsContext.js

```diff
@@ -62,7 +62,7 @@ const defaultContext = [
   () => {},
 ];
 
-const ObservationsContext = React.createContext<ObservationsContextType>(
+const ObservationsContext: React.Context<ObservationsContextType> = React.createContext<ObservationsContextType>(
```

Comment:
Cool, I didn't know you could type it this way

Reply:
yeah it's kinda redundant and definitely not necessary once we convert to TS (since its inference will work out of the box)